linetools/isgm/abssystem.py

```diff
@@ -215,17 +234,26 @@ def __init__(self, radec, zabs, vlim, zem=0., abs_type=None,
         self.flag_ZH = 0
 
         # Abundances and Tables
-        self._EW = QTable()
+        self._EW = Table()
         self._ionN = None   # Needs to be None for fill_ion
-        self._trans = QTable()
+        self._trans = Table()
         self._ionstate = {}
-        self._abund = QTable()
+        self._abund = Table()
 
         # Refs (list of references)
         self.Refs = []
 
+    @property
+    def vlim(self):
+        return self.limits.vlim
+
+    @property
+    def zabs(self):
```

Comment:
could it be cales zsys instad?

Reply:
It probably should have been, but we've been using
zabs.  I worry taking this away would break quite
a bit of code..